types/discord.ts

```diff
@@ -1385,31 +1385,31 @@ export interface DiscordAuditLog {
   webhooks: DiscordWebhook[];
   /** List of users found in the audit log */
   users: DiscordUser[];
-  /** List of audit log entries */
+  /** List of audit log entries, sorted from most to least recent */
   audit_log_entries: DiscordAuditLogEntry[];
   /** List of partial integration objects */
   integrations: Partial<DiscordIntegration>[];
-  /** List of threads found in the audit log. */
+  /** List of threads found in the audit log. * Threads referenced in THREAD_CREATE and THREAD_UPDATE events are included in the threads map since archived threads might not be kept in memory by clients. */
```

Comment:
```suggestion
  /** 
   * List of threads found in the audit log.
   * Threads referenced in `THREAD_CREATE` and `THREAD_UPDATE` events are included in the threads map since archived threads might not be kept in memory by clients. 
   */
```